test/obb/server/controllers/mock_request.clj

```diff
@@ -0,0 +1,12 @@
+(ns obb.server.controllers.mock-request
+  (:use clojure.test)
+  (:require [obb.server.controllers.index :as index]
+            [obb.server.core :as core]
+            [clojure.edn :as edn]
+            [ring.mock.request :as mock]))
+
+(defn http-get
+  "Mocks a HTTP GET request. Returns the body as edn"
+  [path]
+  (let [response (core/app (mock/request :get "/"))]
+    (assoc response :body (edn/read-string (:body response)))))
```

Comment:
Oh yeah, I did not remember about authenticated requests. I never did this but bypassing authentication probably requires to mock the authentication mechanism, right? I cannot see very clearly now why we would need a different function for that, but probably you are right. :+1: